Source/WebCore/platform/graphics/FontSizeAdjust.h

```diff
@@ -69,12 +69,23 @@ struct FontSizeAdjust {
             : std::nullopt;
     }
 
+    bool resolveFromFontIfNeeded(float computedSize, const FontMetrics& fontMetrics)
+    {
+        if (!shouldResolveFromFont())
+            return false;
+        value = computeValue(computedSize, fontMetrics);
+        hasFromFontResolved = true;
```

Comment:
Once you assign `value` you don't need the separate `bool`. That's the whole point of `std::optional` (and therefore `Markable`).